### ISSUE TITLE
Repeating words

### ISSUE THREAD

**Issue by MrVallentin** [KhronosGroup/Vulkan-Docs#148]
Inspired by issue #146, I decided to dig a little and look for more typos. To ease the search, I created a small [Python script and used some regex](https://gist.github.com/MrVallentin/3ceff0853a153356dd70).

There (currently) seems to be a lot of "be be" in [section 30.2.](https://www.khronos.org/registry/vulkan/specs/1.0/xhtml/vkspec.html#features-limits)
and a single "because because" in [section 18.3.](https://www.khronos.org/registry/vulkan/specs/1.0/xhtml/vkspec.html#copies-images)

*Here are the results:*

```
Ln: 15365, Span: (61, 76), Content: 'because because'
Line: compatible with <code class="code">VK_FORMAT_R32_UINT</code> because because both texels are 4
 
Ln: 22891, Span: (52, 57), Content: 'be be'
Line:     buffers that <span class="normative">can</span> be be included in descriptor bindings in a pipeline
 
Ln: 22908, Span: (52, 57), Content: 'be be'
Line:     buffers that <span class="normative">can</span> be be included in descriptor bindings in a pipeline
 
Ln: 22917, Span: (68, 73), Content: 'be be'
Line:     dynamic storage buffers that <span class="normative">can</span> be be included in descriptor bindings
 
Ln: 22925, Span: (51, 56), Content: 'be be'
Line:     images that <span class="normative">can</span> be be included in descriptor bindings in a pipeline
 
Ln: 22937, Span: (51, 56), Content: 'be be'
Line:     images that <span class="normative">can</span> be be included in descriptor bindings in a pipeline
 
Ln: 22946, Span: (56, 61), Content: 'be be'
Line:     attachments that <span class="normative">can</span> be be included in descriptor bindings in a
```

**Comment by oddhack** [comment]
Thanks! I'll fix these in the next public update.

**Comment by oddhack** [comment]
Should be fixed in the 1.0.7 spec update.
